DTS-STN/Alpha-Site
# Update libraries

# Description

[Update libraries](https://trello.com/c/u6p4dgUb/208-208-update-libraries)

Updated every library

## Acceptance Criteria

Update libraries and make sure everything is working

## Test Instructions

1. Test website
2. Test storybooks
3. Try to run the tests
4. Make sure everything is working globally

## Help Requested
I couldn't update to webpack 5 because it makes storybooks crash.

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
